custodian/qchem/jobs.py

```diff
@@ -165,6 +178,11 @@ def opt_with_frequency_flattener(
             check_connectivity (bool): Whether to check differences in connectivity
                 introduced by structural perturbation. Defaults to True.
             linked (bool): Whether or not to use the linked flattener. Defaults to True.
```

Comment:
Can you add a little more detailed description of what the linked flattener is?

Reply:
Done.